enterprise/__init__.py

```diff
@@ -4,6 +4,6 @@
 
 from __future__ import absolute_import, unicode_literals
 
-__version__ = "0.69.1"
+__version__ = "0.69.2"
```

Comment:
@mushtaqak generally we add version update related changes after we have verified our changes on our combined business sandbox (http://business.sandbox.edx.org/) and just before we are going to merge. This make us avoid unnecessary conflicts for version updates and all enterprise team members use this sandbox.

Reply:
As part of version update we will also need to add changes in https://github.com/edx/edx-enterprise/blob/master/CHANGELOG.rst

Reply:
Right on, I will make sure this happens